src/Test.FunctionalTests.Sarif/Multitool/BaselineOptionTests.cs

```diff
@@ -23,6 +23,10 @@ public class BaselineOptionTests : FileDiffingFunctionalTests
     {
         private bool IsInline = false;
 
+        public static readonly TestAssetResourceExtractor BaselineNamespaceExtractor =
```

Comment:
>BaselineNamespaceExtractor [](http://example.com/codeflow?start=58&length=26)

Basic pattern, a test can allocate a 'test resource extractor' that retrieves test data from the resource table. We work against the resource table because it minimizes disk activity and improves tests performance. Probably, a standard test resource extractor could be allocated by a test base class, but that's a refactoring for another day, this change is already huge.